core/src/main/java/de/danoeh/antennapod/core/storage/DBTasks.java

```diff
@@ -298,11 +299,14 @@ public static synchronized Feed updateFeed(Context context, Feed newFeed, boolea
             Log.d(TAG, "Found no existing Feed with title "
                             + newFeed.getTitle() + ". Adding as new one.");
 
-            // Add a new Feed
-            // all new feeds will have the most recent item marked as unplayed
-            FeedItem mostRecent = newFeed.getMostRecentItem();
-            if (mostRecent != null) {
-                mostRecent.setNew();
+            if (!UserPreferences.getSkipInboxSetting()) {
+                // All new feeds will have the most recent item placed in the
+                // Inbox (i.e. set as New), unless the "Skip Inbox" pref is
+                // turned on
+                FeedItem mostRecent = newFeed.getMostRecentItem();
+                if (mostRecent != null) {
+                    mostRecent.setNew();
+                }
```

Comment:
I don't remember exactly what we talked about on the community call, but I think we decided to remove this entirely and never add episodes of a new subscription to the inbox. @keunes do you remember?

Reply:
Yep, I was going to ask about this to confirm, because I thought I remembered this.

Reply:
This has been removed in my most recent push.